### DISCUSSION TITLE
How to export to gif file with transparent background?

### DISCUSSION THREAD

**Discussion by Social-Mean** [3b1b/manim#1849] [Q&A]
I want to insert a transparent gif file to my slides (ppt file) made by manim, but I do not know how to export.

I have used `-ti`, for `t` means 'transparent' and `i` means 'gif', but it does not work.

Thanks!

---

我想将 manim 做的动画导出为透明底的 gif 文件, 以便于插入到 ppt 格式的幻灯片中, 但是我不知道如何导出.

我已经试过 `-ti` 选项, 其中 `t` 代表 '透明', `i` 代表 'gif', 但是没能成功.

谢谢!

**Comment by hunter1749** [comment]
Did you find a way?
